src/routes/connect/ui/wallets/Albedo.ts

```diff
@@ -29,6 +31,18 @@ export default class Albedo extends AbstractWallet implements IWallet {
         super.connectWithWallet(Albedo.NAME, publicKey);
     }
 
+    getConnectObject(): IConnectObject {
+        return {
+            name: 'Albedo',
+            connectMethod: async () => {
+                this.logIn(await this.getPublicKey());
+            },
+            img: albedo,
+            width: 35,
+            height: 45,
```

Comment:
not a wallet object concern, this is part of the component that renders this wallet I'd say